app/controllers/sessions_controller.rb

```diff
@@ -1,11 +1,18 @@
 # frozen_string_literal: true
 
 class SessionsController < Devise::SessionsController
+  prepend_before_action :check_recaptcha, only: [:create]
+  before_action :set_recaptcha, only: [:new]
+
   def create
     super
     set_user_locale if user_signed_in?
   end
 
+  def new
+    super
+  end
```

Comment:
Rubocop required we specify `new` since we have a `before_action` associated with it